e2e/README.md

```diff
@@ -0,0 +1,109 @@
+# E2E Tests
+
+This directory contains an E2E test suite using the [Playwright](https://playwright.dev)
+framework. 
+
+## Getting Started
+
+Please ensure that you have followed the basic installation guide in the
+[root README](./../README.md).
+Once everything is set up,
+
+```sh
+# Run all e2e tests:
+yarn test-e2e 
+```
+
+By default, the test suite will run against the http://localhost:8080.
+You can override this by exporting `E2E_UI_BASE_URL` and `E2E_WS_BASE_URL` environment variables beforehand:
+
+```sh
+# Ex: Set the server URL to dev3:
+export E2E_UI_BASE_URL=https://dev3.openmrs.org/openmrs/spa/
+export E2E_WS_BASE_URL=https://dev3.openmrs.org/openmrs/ws/
+
+# Run all e2e tests:
+yarn test-e2e
+```
+To run a specific test by title:
+```sh
+yarn test-e2e -g "title of the test"
+```
+Check [this documentation](https://playwright.dev/docs/running-tests#command-line) for more running options.  
+
+It is also highly recommended to install the companion VS Code extension:
+https://playwright.dev/docs/getting-started-vscode
+
+
+## Writing New Tests
+
+In general, it is recommended to read through the official [Playwright docs](https://playwright.dev/docs/intro)
+before writing new test cases. The project uses the official Playwright test runner and,
+generally, follows a very simple project stucture:
+
+```
+e2e
+|__ commands
+|   ^ Contains "commands" (simple reusable functions) that can be used in test cases/specs.
```

Comment:
Would be good to have an example here, i.e., what is the kind of simple, reusable function? How does it differ from fixtures, core, pages, etc.